charts/lighthouse/values.yaml

```diff
@@ -275,6 +275,9 @@ tektoncontroller:
   # tektoncontroller.terminationGracePeriodSeconds -- Termination grace period for tekton controller pods
   terminationGracePeriodSeconds: 180
 
+  # tektoncontroller.enableRerunStatusUpdate -- Enable updating the rerun status of PipelineRuns on GitHub
```

Comment:
"rerun status"? Maybe you mean

```suggestion
  # tektoncontroller.enableRerunStatusUpdate -- Enable updating the status on GitHub when PipelineRuns are rerun
```

Reply:
yes it's better explained this way thank you

it's edited